tests/test_sudoku.py

```diff
@@ -60,3 +97,11 @@ def test_sudoku_solver_hard_task():
     s = Sudoku(HARD_TASK)
     s.solve()
     assert s.get_grid() == HARD_SOLUTION
+
+
+def test_sudoku_EEA():
```

Comment:
название автотеста обычно содержит название тестируемого метода.  Так легче читать логи тестирования. А слово `sudoku` уже содержится в названии модуля.
```suggestion
def test_exclude_equal_alternatives():
```